tools/src/icon4py/tools/py2fgen/template.py

```diff
@@ -526,6 +517,12 @@ def get_sizes_maker(arg: FuncParameter) -> str:
    {%- for arr in optional_arrays %}
        !$acc host_data use_device({{ arr }}) if(associated({{ arr }}))
    {%- endfor %}
+   
+   #ifdef _OPENACC
```

Comment:
Stupid question: is this pre-processor flag icon specific or does it come with openacc?

Reply:
It's defined in the standard.